libmscore/ottava.cpp

```diff
@@ -145,53 +145,61 @@ Sid Ottava::getPropertyStyle(Pid pid) const
       {
       Q_ASSERT(int(OttavaType::OTTAVA_22MB) - int(OttavaType::OTTAVA_8VA) == 5);
 
-      static const Sid ss[24] = {
+      static const Sid ss[] = {
             Sid::ottava8VAPlacement,
             Sid::ottava8VAnoText,
+            Sid::ottava8VAnoContinueText,
             Sid::ottava8VBPlacement,
             Sid::ottava8VBnoText,
+            Sid::ottava8VBnoContinueText,
             Sid::ottava15MAPlacement,
             Sid::ottava15MAnoText,
+            Sid::ottava15MAnoContinueText,
             Sid::ottava15MBPlacement,
             Sid::ottava15MBnoText,
+            Sid::ottava15MBnoContinueText,
             Sid::ottava22MAPlacement,
             Sid::ottava22MAnoText,
+            Sid::ottava22MAnoContinueText,
             Sid::ottava22MBPlacement,
             Sid::ottava22MBnoText,
+            Sid::ottava22MBnoContinueText,
 
             Sid::ottava8VAPlacement,
             Sid::ottava8VAText,
+            Sid::ottava8VAContinueText,
             Sid::ottava8VBPlacement,
             Sid::ottava8VBText,
+            Sid::ottava8VBContinueText,
             Sid::ottava15MAPlacement,
             Sid::ottava15MAText,
+            Sid::ottava15MAContinueText,
             Sid::ottava15MBPlacement,
             Sid::ottava15MBText,
+            Sid::ottava15MBContinueText,
             Sid::ottava22MAPlacement,
             Sid::ottava22MAText,
+            Sid::ottava22MAContinueText,
             Sid::ottava22MBPlacement,
             Sid::ottava22MBText,
+            Sid::ottava22MBContinueText,
             };
 
+      int idx = int(_ottavaType) * 3 + (_numbersOnly ? 0 : (sizeof(ss)/sizeof(*ss))/2);
       switch (pid) {
             case Pid::OFFSET:
                   return placeAbove() ? Sid::ottavaPosAbove : Sid::ottavaPosBelow;
-            case Pid::PLACEMENT: {
-                  int idx = int(_ottavaType) * 2 + (_numbersOnly ? 0 : 12);
+            case Pid::PLACEMENT:
                   return ss[idx];
```

Comment:
would those need to get changed too, so something like `return ss.at(idx);`?

Reply:
Or is that optional?

Reply:
Vector elements can be accessed by `operator[]`, so there is nothing to change here.

Reply:
OK, so optional.